### PR TITLE
Add simple player poc

### PR DESCRIPTION
**Pull request by novayo**
Issue: https://github.com/novayo/Mario_cplus/issues/10

### CODE REVIEW

File: examples/simple_player_poc.cc [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    SDL_Renderer* surface = sdl_handler::GetInstance().get_surface();
+    int close = 0;
+    SDL_Event event;
+
+    // Init player and blocks
+    printf("Generate backgroud and player...\n");
+    Block player(400, 280, 255, 255, 0);
+    std::vector<Block> bg_blocks;
+    for (int j = 0; j <= NUM_ROW; j++) {
+        bool draw_red = j % 2 == 0;
```
Comment (tkwang0530) [inline examples/simple_player_poc.cc:63]:
```suggestion
        bool draw_black = j % 2 == 0;
```

File: examples/simple_player_poc.cc [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    Mix_OpenAudio(44100, MIX_DEFAULT_FORMAT, 2, 1024);
+    Mix_Music* background_music = Mix_LoadMUS("../res/sounds/overworld.wav");
+    Mix_PlayMusic(background_music, -1);
+
+    // Load sound effect
+    Mix_Chunk* sound_effect = Mix_LoadWAV("../res/sounds/coin.wav");
+
+    // Init sdl_handler
+    sdl_handler::GetInstance().initialize();
+    SDL_Renderer* surface = sdl_handler::GetInstance().get_surface();
```
Comment (tkwang0530) [inline examples/simple_player_poc.cc:54]:
```suggestion
```

Comment (tkwang0530) [reply examples/simple_player_poc.cc:54]: unused line

File: examples/simple_player_poc.cc (hunk 1 above)
Comment (novayo) [reply examples/simple_player_poc.cc:63]: Done.

File: examples/simple_player_poc.cc (hunk 2 above)
Comment (novayo) [reply examples/simple_player_poc.cc:54]: Done.

**Review by tkwang0530 — APPROVED** [review]

**Pull request merged**